logbook-api/src/main/java/org/zalando/logbook/RequestURI.java

```diff
@@ -63,13 +75,11 @@ private static String reconstruct(final HttpRequest request, final Set<Component
         if (components.contains(QUERY) && !query.isEmpty()) {
             url.append('?').append(query);
         }
-
-        return url.toString();
     }
 
     private static boolean isNotStandardPort(final String scheme, final int port) {
-        return "http".equals(scheme) && port != 80 ||
-                "https".equals(scheme) && port != 443;
+        return ("http".equals(scheme) && port != 80) ||
```

Comment:
Those are not really needed, are they?

Reply:
Strictly, no, it was intended as an improvement in readability, forgot to comment.